### PR TITLE
Add DeepAR ap-northeast-3 information.

### PR DESCRIPTION
**Pull request by jaheba**
*Issue #, if available:*

*Description of changes:* Adding missing repository information for DeepAR algorithm in ap-northeast-3.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by jaheba** [comment]
PR for SageMaker-SDK: https://github.com/aws/sagemaker-python-sdk/pull/3120

File: doc_source/ecr-ap-northeast-3.md [hunk 1]
```diff
@@ -97,6 +98,21 @@ image_uris.retrieve(framework='debugger',region='ap-northeast-3')
...
+
+```
+from sagemaker import image_uris
+image_uris.retrieve(framework='forecasting-deepar',region='ap-northeast-3')
+```
+-->
+
+| Registry path | Version | Job types \(image scope\) | 
+| --- | --- | --- | 
+| 867004704886\.dkr\.ecr\.ap\-northeast\-2\.amazonaws\.com/forecasting\-deepar:<tag> | 1 | inference, training |
```
Comment (jgasthaus) [inline doc_source/ecr-ap-northeast-3.md:114]: Should this be ap-northeast-3 (not 2)?

Comment (jaheba) [reply doc_source/ecr-ap-northeast-3.md]: Thanks! Updated the PR.

**Review by jgasthaus — APPROVED** [review]

**Comment by judyheflin** [comment]
Thanks for bringing this to our attention! This image is supported and is public on the SageMaker developer guide.

**Pull request merged**